.changeset/real-buses-pay.md

```diff
@@ -0,0 +1,10 @@
+---
+'@twilio-paste/button': patch
+'@twilio-paste/core': patch
+'@twilio-paste/menu': patch
+---
+
+- [Button] Update destructive Buttons except destructive link Buttons to use colorTextDestructive
+- [Button] Update destructive Button style tests
+- [Menu] Update destructive Menu items to use colorTextDestructive
+- [Menu] Add style tests for default and destructive Menu items
```

Comment:
Unless it's the exact same changelog message for multiple packages, we should split these up into a `menu + core` changeset and a `button + core` changeset individually.

Reply:
We don't mention tests in changelogs normally since it isn't valuable for users to know in relation to version bumps, since tests dont cause version bumps.